packages/babel-parser/src/parser/statement.js

```diff
@@ -1197,8 +1195,7 @@ export default class StatementParser extends ExpressionParser {
     this.expect(tt.braceL);
 
     // For the smartPipelines plugin: Disable topic references from outer
-    // contexts within the class body. They are permitted in test expressions,
-    // outside of the class body.
+    // contexts within the class body.
```

Comment:
I'm not sure this shortening is an improvement.

Reply:
I'm believe the existing comment was copy-pasted from `parseWhileStatement`, where the mention of a "test expression" makes sense since `while` loops have one, i.e.
```while (<test_expression>) {```
but since `class` bodies do not, removing mention of "test expression" is removing objectively false information from the comment (which I found to be quite confusing when I first encountered the comment).

Reply:
Ah, fair.